201-zerto-zca-vpn/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{  
+   "$schema":"http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{      
+     "localGatewayIpAddress":{  
```

Comment:
put these in as defaultValues in the template where possible - i.e. anything not GEN'd

Reply:
done